HPCC-15187 DOCS:Options dropdown in QuerySetQueryWidget

g-pan opened the pull request:
FIX HPCC-15187 DOCS:Options dropdown in QuerySetQueryWidget
Update the ECL Watch manual the Queries section in particular
to show not only the Options action button, but to document
all the action buttons on the queries page/tab, as none were
previously documented.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com
@JamesDeFabia please review
@miguelvazq please review

g-pan committed:
HPCC-15187 DOCS:Options dropdown in QuerySetQueryWidget

FIX HPCC-15187 DOCS:Options dropdown in QuerySetQueryWidget
Update the ECL Watch manual the Queries section in particular
to show not only the Options action button, but to document
all the action buttons on the queries page/tab, as none were
previously documented.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest 
sha: 9403b84d6642f2df57f3cd5236b9e39ef6d94f66
Build: success
Install hpccsystems-platform-community_6.0.0-beta1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -158,6 +158,148 @@ depending on where they wind up.-->
+                </itemizedlist>
+
+                <para>The Filter also also supports wild cards. To limit or
+                filter your search results you can use keywords as displayed
```
Remove the second sentence, keywords do not apply.

miguelvazq commented:
The screenshots look current.

g-pan committed:
HPCC-15187 DOCS:Options dropdown in QuerySetQueryWidget

FIX HPCC-15187 DOCS:Options dropdown in QuerySetQueryWidget
Update the ECL Watch manual the Queries section in particular
to show not only the Options action button, but to document
all the action buttons on the queries page/tab, as none were
previously documented.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented:
@g-pan Looks Good to merge now

HPCCSmoketest commented:
Automated Smoketest 
Sha: 93e6b2b4356e5ebe725e5a61e483b101a60f0088
Build: success
Install hpccsystems-platform-community_6.0.0-beta1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

g-pan commented:
good to merge.
